### ISSUE TITLE
Module parse failed: Unexpected token

### ISSUE THREAD

**Issue by nspeedzy** [eslint/eslint#13445]
**Description**
I am creating a shopping-cart app in vuejs where i am using typescript as the main language. But then, when i try to compile the project, It gives me a parsing error, unexpected token `:`. By the way, i am declaring variables with type annotations. But, Is there anything i am missing here? What should i do to fix this? 

**Tell us about your environment**

* **ESLint Version:** 6.8.0
**Node Version:** 13.8.0
* **npm Version:** 6.13.6

**What parser (default, Babel-ESLint, etc.) are you using?**
Babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

`.eslintrc.json`
```json
{
    "root": true,
    "env": {
        "node": true
    },
    "extends": [
        "plugin:vue/essential",
        "eslint:recommended"
    ],
    "parserOptions": {
        "parser": "babel-eslint"
    },
    "rules": {}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```ts
import Vue from 'vue'
var store : string;
store = "Hello world";
alert(store);
//
export default Vue.extend({
    name: 'App',
    data() {
        return {
            num1: 0,
            num2: 0
        };
    }
})
```
I used the following command to lint the project. (Vuejs uses eslint here)

```bash
vue-cli-service lint
```

**What did you expect to happen?**
No parsing errors should have occured, especially like the unexpected token errors

**What actually happened? Please include the actual, raw output from ESLint.**
The eslint gave me a parsing error. Code:
```log
Module parse failed: Unexpected token (20:7)
File was processed with these loaders:
 * ./node_modules/cache-loader/dist/cjs.js
 * ./node_modules/vue-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|
| import Vue from 'vue'
> var ts : string;
| ts = "Hello world";
| alert(ts);

 @ ./src/components/App.vue?vue&type=script&lang=ts& 1:0-172 1:188-191 1:193-362 1:193-362
 @ ./src/components/App.vue
 @ ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/Index.vue?vue&type=script&lang=js&
 @ ./src/Index.vue?vue&type=script&lang=js&
 @ ./src/Index.vue
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://192.168.1.3:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.js
```

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by kaicataldo** [comment]
This looks like a Webpack error, not an ESLint error.

**Comment by nspeedzy** [comment]
Ok thanks! I figured it out! In the Vue CLI, i forgot to add typescript language support for my vuejs app
